eureka2-integration/src/test/java/com/netflix/eureka2/integration/server/startup/BridgeServerStartupAndShutdownIntegrationTest.java

```diff
@@ -30,13 +36,15 @@
     public void setUpBridgeServer() throws Exception {
         bridgeServer = new EurekaBridgeServer(SERVER_NAME) {
             @Override
-            protected AbstractModule createEureka1ClientModule() {
-                return new AbstractModule() {
-                    @Override
-                    protected void configure() {
-                        bind(DiscoveryClient.class).toInstance(eureka1ServerResource.createDiscoveryClient("bridgeService"));
-                    }
-                };
+            protected Module getModule() {
+                return Modules.override(super.getModule())
+                        .with(new AbstractModule() {
+                            @Override
+                            protected void configure() {
+                                bind(EurekaInstanceConfig.class).toProvider(MyDataCenterInstanceConfigProvider.class).in(Scopes.SINGLETON);
```

Comment:
There is an upstream change where we install(EurekaModule) instead of just binding DiscoveryClient, which necessitates this here.